pkg/kudoctl/bundle/finder/bundle_finder_test.go

```diff
@@ -0,0 +1,46 @@
+package finder
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestManager_GetBundle(t *testing.T) {
+	f := &Manager{
```

Comment:
Whenever I see something named like this, my mind immediately goes to https://stackoverflow.com/questions/1866794/naming-classes-how-to-avoid-calling-everything-a-whatevermanager :)